generator/osm2meta.cpp

```diff
@@ -337,6 +337,17 @@ std::string MetadataTagProcessorImpl::ValidateAndFormat_local_ref(std::string co
   return v;
 }
 
+std::string MetadataTagProcessorImpl::ValidateAndFormat_smoking(std::string v)
+{
+  strings::AsciiToLower(v);
+  if (v == "outside" || v == "yes" || v == "no")
+    return v;
+  // Process additional top tags.
+  if (v == "separated" || v == "isolated" || v == "dedicated")
+    return "yes";
```

Comment:
I mean keep outside, yes, no, separated, isolated, dedicated values as-is.

But keep only 3 options when showing the status in the Place Page (not in this PR).